.gitignore

```diff
@@ -149,5 +149,4 @@ htmlcov/
 # IDEs
 .idea
 
-.env*
-!.env.sample
+.env
```

Comment:
yeah I don't think we'll ever have anything other than a .env locally? If so I think we can just ignore those explicitly because the !.env.sample / !.env.example doesn't work